write_compositions.py

```diff
@@ -38,15 +47,6 @@
 else:
     print("No job queue is setup. Running in serial mode here rather than on the cluster")
 
-    # setup CSV file and write header
-    f = open(os.path.join(output_dir, 'comp_mass_output.csv'),'w',newline='')
-    # write header
-    header = ['Run ID','MOF','Mass']
-    for gas in gases:
-        header.append(gas)
-    writer = csv.writer(f, delimiter='\t')
-    writer.writerow(header)
-
```

Comment:
the f.close() below should be up a level so that it gets properly closed even if you are queueing on the job server.